Fix EPUBs breaking when sent to Kindle

f0lie opened the pull request:
The README.md was also updated to include possible future features and some known shortcomings of the files.

f0lie committed:
Move markdown files into seperate folder.

f0lie committed:
Clean up function a little bit.

f0lie committed:
Fix deprecation warning.

f0lie committed:
Refactor some code so it's easier to change later.

f0lie committed:
Added meta data block to markdown for pandoc conversion later.

f0lie committed:
Refactored code.

f0lie committed:
Refactor metablock code.

f0lie committed:
Refactor original code for intented markdown files because the epub files don't actually need intentation.

The intention is still useful for just reading the markdown as it is.

f0lie committed:
Added EPUB generation using pandocs.

f0lie committed:
Added submission links to original reddit posts.

f0lie committed:
Merge remote-tracking branch 'upstream/main'

f0lie committed:
Updated README.md

f0lie committed:
Update README.md again

f0lie committed:
Generate pdfs

f0lie committed:
Fixed broken github action because of double quotes within f-string

f0lie committed:
Fixed double quote issue.

f0lie committed:
Fix readme.md

f0lie committed:
Update github action to use latest python version.

f0lie committed:
Ignore .secrets file that store information for local github workflow runs.

f0lie committed:
Added pdf generation and updated the workflow to be up to date.

The .gitignore was updated to ignore files used to test locally working the workflow using act.

f0lie committed:
Updated git push command to be more verbose for debugging later.

f0lie committed:
Fixed problem where venv was being ignored by the python script because each run uses a different shell in github workflow.

f0lie committed:
Add sudo back into workflow.

f0lie committed:
Added doc about running github workflow locally.

f0lie committed:
Added sudo to the right places to fix workflow.

actions-user committed:
Update Reddit comments and markdown files

f0lie committed:
Merge branch 'uruvelakassapa:main' into main

f0lie committed:
Update README.md

f0lie committed:
Update README.md

f0lie committed:
Update README.md

f0lie committed:
Update README.md

f0lie committed:
Fixed epubs breaking when sent to Kindle. #1

f0lie committed:
Refactor global cursor variable out.

f0lie committed:
Simplify logic behind md generation.

f0lie committed:
Use Path objects instead of plain strings for paths.

f0lie committed:
Improve error handling.

actions-user committed:
Update Reddit comments and markdown files

f0lie committed:
Add unit tests for generate_markdown and fix various bugs.

uruvelakassapa commented:
@f0lie There are some conflicts at the moment. But I think it might be easier if you continue development on your fork and I'll link to it in my readme. My main aim with this repo was to just have a backup of Venerable Anīgha's posts in case something happened to reddit.

f0lie committed:
Add coverage into the required packages.

f0lie committed:
Add documentation for all of the functions of the code.

f0lie committed:
Added pyproject.toml for more sensible builds.

Changed SQL query not depend on *.

Added documentation on README.md.

Removed cache files and updated gitignore.

f0lie committed:
Ignore .jj folder.

f0lie committed:
Fix broken update_comments.yml.

actions-user committed:
Update Reddit comments and markdown files

f0lie committed:
Fix pytest config.

f0lie committed:
Refactor fetch_comments.py.

f0lie committed:
Removed artifact actions because artifacts only exist in the same workflow.

f0lie committed:
Migrated to using uv.

actions-user committed:
Update Reddit comments and markdown files

actions-user committed:
Update Reddit comments and markdown files
